.github/workflows/build.yaml

```diff
@@ -286,14 +289,14 @@ jobs:
             --volname "Freeciv21 Game Installer" \
             --volicon "client.icns" \
             --window-pos 200 120 \
-            --window-size 800 400 \
+            --window-size 600 400 \
             --icon-size 128 \
             --icon "Freeciv21.app" 200 190 \
             --hide-extension "Freeciv21.app" \
-            --app-drop-link 600 185 \
-            "Freeciv21-${{steps.split.outputs.fragment}}.dmg" \
+            --app-drop-link 400 185 \
+            "Freeciv21-${{steps.split.outputs.fragment}}-arm64-osx.dmg" \
             "Freeciv21.app"
-          shasum -a 256 Freeciv21-${{steps.split.outputs.fragment}}.dmg > Freeciv21-${{steps.split.outputs.fragment}}.dmg.sha256
+          shasum -a 256 Freeciv21-${{steps.split.outputs.fragment}}-arm64-osx.dmg > Freeciv21-${{steps.split.outputs.fragment}}-arm64-osx.dmg.sha256
       - name: Upload package
         uses: softprops/action-gh-release@v2
         if: startsWith(github.ref, 'refs/tags/')
```

Comment:
Will want to restrict this to one of the jobs.

Reply:
Can correct at a later date